awx/ui/src/screens/Instances/InstanceAdd/InstanceAdd.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Card, PageSection } from '@patternfly/react-core';
+import { InstancesAPI } from 'api';
+import InstanceForm from '../Shared/InstanceForm';
+
+function InstanceAdd() {
+  const history = useHistory();
+  const [formError, setFormError] = useState();
+  const handleSubmit = async (values) => {
+    const { instanceGroups, executionEnvironment } = values;
+    values.execution_environment = executionEnvironment?.id;
+
+    try {
+      const {
+        data: { id },
+      } = await InstancesAPI.create(values);
+
+      for (const group of instanceGroups) {
+        await InstancesAPI.associateInstanceGroup(id, group.id);
```

Comment:
spoke with @jbradberry and I think we want to allow default behavior here. E.g. the user should be able to create an Instance and not associate it with any instance group. They can always associate it later in the InstancesGroup list page.